lv_conf_template.h

```diff
@@ -488,6 +488,8 @@ typedef void * lv_obj_user_data_t;
 
 /*Gauge (dependencies:lv_bar, lv_linemeter)*/
 #define LV_USE_GAUGE    1
+/*Type for gauge min, max, value*/
+typedef int16_t lv_gauge_value_t;
```

Comment:
For simplicity, we can use `int32_t` directly  instead of `lv_gauge_value_t`. It means only a few bytes overhead in gauges and line meters. Typically there is only a 1-2 from this type of objects so the overhead is negligible. IMO the less config options are more important.